ISISComputingGroup/ibex_gui
# Script Server Perspective

Added some placeholder UI for the script server, as per https://github.com/ISISComputingGroup/IBEX/issues/1363.

This is in no way meant to be a final version, just an idea to get the ball rolling. Currently the perspective is described as NICOS, this shouldn't be permanent as we don't want to tie everything to a specific back-end

To test:
* Confirm NICOS perspective appears on switcher
* Confirm NICOS view makes some sense
* Confirm 'Create Script' button creates a view that makes some sense
